apis/ec2/v1alpha1/zz_types.go

```diff
@@ -1711,8 +1955,9 @@ type ManagedPrefixList struct {
 	Tags []*Tag `json:"tags,omitempty"`
 }
 
+// +kubebuilder:skipversion
 type ModifyVPNTunnelOptionsSpecification struct {
-	DPDTimeoutAction *string `json:"dPDTimeoutAction,omitempty"`
+	DPDTimeoutAction *string `json:"dpdTimeoutAction,omitempty"`
```

Comment:
Another breaking change.

Reply:
It seems like we don't generate the CRD that was supposed to use this struct, hence it doesn't show up in any CRDs.